generators/database-changelog-liquibase/templates/src/main/resources/config/liquibase/changelog/added_entity.xml.ejs

```diff
@@ -73,19 +73,19 @@
         <%_ for (relationship of joinTableRelations) { _%>
 
         <createTable tableName="<%= relationship.joinTable.name %>">
-            <%_ for (field of relationship.otherEntity.primaryKey.fields) { _%>
```

Comment:
is `ownFields` and `fields` availabel from `ids`?

Reply:
Please read comment above.

Reply:
I don't know if answered your question, since I don't know if I understand it correctly, hopefully yes.